notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb

```diff
@@ -0,0 +1,681 @@
+    "                processing_time = np.mean(processing_times) * 1000\n",
+    "                fps = 1000 / processing_time\n",
+    "\n",
+    "            # Visualize the results\n",
+    "            for i in range(0, 3):\n",
+    "                display_text = text_template.format(\n",
+    "                    label=decoded_labels[i],\n",
+    "                    conf=decoded_top_probs[i] * 100,\n",
+    "                )\n",
+    "                display_text_fnc(frame, display_text, i)\n",
+    "\n",
+    "            display_text = text_inference_template.format(Time=processing_time, fps=fps)\n",
+    "            display_text_fnc(frame, display_text, 3)\n",
+    "\n",
+    "            # Use this workaround you experience flickering\n",
+    "            if use_popup:\n",
+    "                cv2.imshow(title, frame)\n",
+    "                key = cv2.waitKey(1)\n",
+    "                # escape = 27\n",
+    "                if key == 27:\n",
+    "                    break\n",
+    "            else:\n",
+    "                # Encode numpy array to jpg\n",
+    "                _, encoded_img = cv2.imencode(\".jpg\", frame, params=[cv2.IMWRITE_JPEG_QUALITY, 90])\n",
+    "                # Create IPython image\n",
+    "                i = display.Image(data=encoded_img)\n",
+    "                # Display the image in this notebook\n",
+    "                display.clear_output(wait=True)\n",
+    "                display.display(i)\n",
+    "\n",
+    "    # ctrl-c\n",
+    "    except KeyboardInterrupt:\n",
+    "        print(\"Interrupted\")\n",
+    "    # Any different error\n",
+    "    except RuntimeError as e:\n",
+    "        print(e)\n",
+    "    finally:\n",
+    "        if player is not None:\n",
+    "            # stop capturing\n",
+    "            player.stop()\n",
+    "        if use_popup:\n",
+    "            cv2.destroyAllWindows()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run Action Recognition on a Video File\n",
+    "\n",
+    "Now let's see how the model works in a video file. [Any format supported](https://docs.opencv.org/4.5.1/dd/d43/tutorial_py_video_display.html) by OpenCV will work. Anytime you can press the stop button while the video file is running, it will activate the webcam for the next step.\n",
+    "\n",
+    "> Note: _Sometimes, the video could be cut off if there are corrupted frames. In that case you could convert. If you experience any problems with your video, please use the [HandBrake](https://handbrake.fr/) and select the MPEG format._"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "video_file = \"https://archive.org/serve/ISSVideoResourceLifeOnStation720p/ISS%20Video%20Resource_LifeOnStation_720p.mp4\"\n",
+    "run_action_recognition(video_file, flip=False, use_popup=False, skip_first_frames=600)"
```

Comment:
You still need to add something like that:
  "test_replace": {
    "skip_first_frames=600": "skip_first_frames=10000"
  },